ESPixelStick/src/service/DisplayOLED.cpp

```diff
@@ -26,38 +26,33 @@
 #include <Adafruit_GFX.h>
 #include <Adafruit_SSD1306.h>
 #include "DisplayOLED.h"
-#define SCREEN_WIDTH 128    // OLED display width, in pixels
-#define SCREEN_HEIGHT 32    // OLED display height, in pixels
-#define OLED_RESET -1       // Reset pin # (or -1 if sharing Arduino reset pin)
-#define SCREEN_ADDRESS 0x3C ///< See datasheet for Address; 0x3D for 128x64, 0x3C for 128x32
+int SCREEN_WIDTH = 128;        // OLED display width, in pixels
+int SCREEN_HEIGHT = 32;        // OLED display height, in pixels
+int OLED_RESET = -1;           // Reset pin # (or -1 if sharing Arduino reset pin)
+byte SCREEN_ADDRESS = byte(0x3C); ///< See datasheet for Address; 0x3D for 128x64, 0x3C for 128x32
 Adafruit_SSD1306 display(SCREEN_WIDTH, SCREEN_HEIGHT, &Wire, OLED_RESET);
 
-void c_OLED::Begin ()
+void c_OLED::Begin()
 {
     // DEBUG_START;
 
     // DEBUG_V(String("OLED Enabled: ") + String("WIDTH: " + SCREEN_WIDTH + " HEIGHT: " + SCREEN_HEIGHT + "ADDRESS: " +SCREEN_ADDRESS);
-   display.begin(SSD1306_SWITCHCAPVCC, SCREEN_ADDRESS);
-
-    // DEBUG_END;
-} // Begin
-
-void c_OLED::Booting ()
-{
+    display.begin(SSD1306_SWITCHCAPVCC, SCREEN_ADDRESS);
```

Comment:
I think I mentioned this before and you indicated that you are using the "default" GPIO I2C values. This implementation needs to use the I2C pins defined in the GPIO files or it will conflict with other devices that may be attached to the ESP.